### PR TITLE
Replace all uses of GETOUT waypoints

### LINKED ISSUE #3374: [Arma bug]: GETOUT waypoint is very broken
The GETOUT waypoint often causes units to leave a vehicle long before the waypoint is reached. In this case, the waypoint also never completes. This seems to have an element of undefined behaviour: In sequential testing with identical commands, odd-numbered vehicles would consistently fail while even-numbered vehicles worked correctly. Open vehicle types may be more susceptible, but the issue seems to be unrelated to enemy detection, and setUnloadInCombat has no effect.

In this case the GETOUT waypoint was being used as a cargo waypoint twinned with a TR UNLOAD on the crew, but I'd already run into problems with GETOUT waypoints on combined crew/cargo groups, so it's time for the cancer to be stripped out entirely. GETOUT waypoints can be replaced with a move waypoint plus setWaypointStatements.

Example replication on Altis with IFA loaded (factions don't matter). Usually disembarks at third waypoint:
`["LIB_US_M3_Halftrack", "Normal", "defence", [], Occupants, "airport_3", markerPos "outpost_18"] call A3A_fnc_createAttackVehicle;`

### PR DESCRIPTION
**Pull request by jaj22**
### What type of PR is this.
1. [X] Arma bug workaround
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
The GETOUT waypoint is very broken and can cause squads to leave their transport vehicle for no good reason. Fortunately it is relatively straightforward to replace, so this PR does so.

Tested all uses except the destroy heli mission. It's probably fine.  

### Please specify which Issue this PR Resolves.
closes #3374

### Please verify the following and ensure all checks are completed.
1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Create land QRFs like this:
```
_destPos = markerPos "Outpost_13";
["QRFLAND", Occupants, "defence", 500, objNull, _destPos, 0.8, -1] call A3A_fnc_createSupport;
```

Or more specific vehicle types and source/dest:
```
_sourceMrk = "Airport_3";
_destPos = markerPos "Outpost_18";
["LIB_US_M3_Halftrack", "Normal", "defence", [], Occupants, _sourceMrk, _destPos] call A3A_fnc_createAttackVehicle;
```

### CODE REVIEW

**Review by ante185 — APPROVED** [review]
Tested a couple vehicles, was expecting the low capacity cars to fail as the driver gets added to the group when the capacity is less than 4.
But it worked regardless

tldr LGTM
